package.json

```diff
@@ -16,10 +16,11 @@
     "url": "https://github.com/feross/standard/issues"
   },
   "dependencies": {
+    "clone-deep": "^0.1.1",
     "commondir": "^1.0.1",
     "dezalgo": "^1.0.1",
     "editorconfig-get-indent": "^1.0.0",
-    "eslint": "0.23.0",
+    "eslint": "https://github.com/eslint/eslint.git#80826e1f58b09bcce9490141896f18138022ecd3",
```

Comment:
I went through all the rules and I figured I already did the work for checking out the newer rules before seeing that they pulled in after 0.23.0